src/admin-views/editor/fieldset/settings-provider.php

```diff
@@ -72,7 +70,7 @@ class="tribe-ticket-editor-field-default_provider settings_field ticket_field"
 						 *
 						 * @param string $module - the name of the module
 						 */
-						echo apply_filters( 'tribe_events_tickets_module_name', esc_html( $module ) );
+						echo apply_filters( 'tribe_events_tickets_module_name', esc_html( $active_provider['name'] ) );
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'apply_filters'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 29e480d13355b4c5e98bfe73d44360be</sub>

Reply:
done